Work around chown being mocked on OpenBSD on `make test`.

Index: tests/fileops/chown.c
--- tests/fileops/chown.c.orig
+++ tests/fileops/chown.c
@@ -1,6 +1,7 @@
 #include <stic.h>
 
-#ifndef _WIN32
+/* Port system of OpenBSD mocks `chown` which breaks this test */
+#if !defined(_WIN32) && !defined(__OpenBSD__)
 
 #include <sys/types.h>
 #include <unistd.h> /* chdir() unlink() */
